bundles/org.openhab.binding.mqtt.awtrixlight/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,593 @@
+		<label>Automatc Brightness</label>
+		<description>Let the clock set brightness automatically based on internal brightness sensor readings</description>
+		<category>screen</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="brightness">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Display Brightness</label>
+		<description>Screen brightness in percent</description>
+		<category>screen</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
+		</tags>
+		<state pattern="%.0f %unit%"></state>
+	</channel-type>
+
+	<channel-type id="rssi">
+		<item-type>Number:Dimensionless</item-type>
+		<label>WiFi Signal</label>
+		<description>RSSI value of WiFi signal</description>
+		<category>qualityofservice</category>
+		<tags>
+			<tag>Measurement</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f"></state>
+	</channel-type>
+
+	<channel-type id="indicator">
+		<item-type>Switch</item-type>
+		<label>Indicator</label>
+		<description>Indicator state</description>
+		<category>switch</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Power</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="app">
+		<item-type>String</item-type>
+		<label>Active App</label>
+		<description>App currently shown on screen</description>
+		<category>screen</category>
+		<tags>
+			<tag>Status</tag>
+		</tags>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="screen">
+		<item-type>Image</item-type>
+		<label>Screen Mirror</label>
+		<description><![CDATA[
+			Mirror screen image from the clock.
+			By default only refreshed on app change. Refresh can be also triggered with a RefreshType command.
+			]]>
+		</description>
+		<category>screen</category>
+		<tags>
+			<tag>Status</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="mainColor">
+		<item-type>Color</item-type>
+		<label>Color</label>
+		<description>Color to display text and charts on the screen</description>
+		<category>colorpicker</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>ColorTemperature</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="reset" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Reset</label>
+		<description>Resets the app and the linked items to the default settings</description>
+		<category>switch</category>
+		<tags>
+			<tag>Control</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="background">
+		<item-type>Color</item-type>
+		<label>Background Color</label>
+		<description>Color for display background</description>
+		<category>colorpicker</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>ColorTemperature</tag>
+		</tags>
+	</channel-type>
+
+	<channel-type id="gradientColor">
+		<item-type>Color</item-type>
+		<label>Gradient Color</label>
+		<description>Color text as gradient from Color to Gradient Color</description>
+		<category>colorpicker</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>ColorTemperature</tag>
```

Comment:
Drop the tags, a gradient is not a colortemperature.

Reply:
Done